python/opaque/__init__.py

```diff
@@ -331,7 +331,7 @@ def RecoverCredentials(resp, sec, cfg, infos, key=None):
     ids.idS = ctypes.cast(ctypes.create_string_buffer(1024), ctypes.c_char_p)
     ids.idS_len=1024
 
-    __check(opaquelib.opaque_RecoverCredentials(resp, sec, key, len(key) if key else 0, ctypes.pointer(cfg), ctypes.pointer(infos) if infos else None, ctypes.pointer(ids), sk, auth, export_key))
+    __check(opaquelib.opaque_RecoverCredentials(resp, sec, key, len(key) if key else 0, None, ctypes.pointer(cfg), ctypes.pointer(infos) if infos else None, ctypes.pointer(ids), sk, auth, export_key))
```

Comment:
setting this to none is incorrect, we want to pass pkS here if it is specified. if you cancel this change, i'll fix it correctly.

Reply:
well, it is correct, but hides the fact that there is a possibility to provide a pkS manually, silencing the problem like this, reduces the urgency of fixing this correctly. btw the same problem also applies to php. better to fix it correctly, than to silence the problem.

Reply:
That's reasonable.

I removed the commit. Feel free to add back running the Python test in the GitHub action after the fix gets in. If you'd like me to do that after you make the fix, let me know.

On a separate note, I also was working on steps to run the PHP tests:
```
$ apt-get install -y php php-dev
$ phpize
$ LIBOPAQUE_CFLAGS='-I ../src' LIBOPAQUE_LIBS='-lopaque' ./configure
$ make EXTRA_CFLAGS=-I../src EXTRA_LDFLAGS=-L../src
$ LD_LIBRARY_PATH=../src ./run-tests.php tests/*.phpt
```